apps/sasquatch/build.gradle

```diff
@@ -68,15 +74,22 @@ dependencies {
 
     projectDependencyImplementation project(':sdk:appcenter-analytics')
     projectDependencyImplementation project(':sdk:appcenter-crashes')
-    projectDependencyImplementation project(':sdk:appcenter-distribute')
     projectDependencyImplementation project(':sdk:appcenter-push')
 
     def appCenterSdkVersion = "3.2.2"
     jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-analytics:${appCenterSdkVersion}"
     jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-crashes:${appCenterSdkVersion}"
-    jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-distribute:${appCenterSdkVersion}"
     jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-push:${appCenterSdkVersion}"
 
+    def taskRequests = getGradle().getStartParameter().getTaskRequests().toString()
+    if (taskRequests.contains("GooglePlay")) {
```

Comment:
You don't need conditions here, just use prefix like in other places

Reply:
We cannot do this, because we have in `productFlavors` there are other products that will have to be prescribed for each dependency.

Reply:
We should still be able to combine projectDependency and appCenter for example.
We should test via dimension specs it as we will probably not document this with a if.

Reply:
I checked how it works with use `projectDependencyAppCenterImplementation` and `projectDependencyGooglePlayImplementation`, but in these cases it is failing with error "Could not find method ..". So using this condition is one way to resolve it.